index.html

```diff
@@ -4479,6 +4479,50 @@ <h2>HTTP</h2>
         <section class="informative">
           <h2>Credential Type-Specific Processing</h2>
 
+          <p>
+RDF [[?RDF11-CONCEPTS]] is a technology for modeling graphs of statements. Each statement is a
+single <i>subject&#x2192;property&#x2192;value</i> (also known as
```

Comment:
Any particular reason we can't use the `→` character here?

Reply:
Presuming we can...
```suggestion
single <i>subject→property→value</i> (also known as
```